docker/ray-ml/install-ml-docker-requirements.sh

```diff
@@ -32,10 +34,13 @@ pip --no-cache-dir install -U \
            -r ray-docker-requirements.txt
 
 # explicitly install (overwrite) pytorch with CUDA support
-pip --no-cache-dir install -U \
+# do not use compiled requirements as constraint here as some
+# device-specific constraints can't be met (e.g. torch+cpu).
+# Instead, we don't use `-U` so other dependencies don't get updated.
```

Comment:
We can probably improve this by replacing `torch+cpu` with `torch+cu118` in a script. LMK what you think, the current version works well